README.md

```diff
@@ -65,6 +65,40 @@ High performance and convenient features are the main goals.
     cargo install --locked --git https://github.com/pamburus/hl.git
     ```
 
+* Run by using [nix](https://nixos.org/download/)
```

Comment:
Could you please let me know why you are using "by using" here instead of "using"?
I am not a native English speaker, but I did some searching on the internet about this, and according to what I [found](https://proofed.com/writing-tips/word-choice-using-vs-by-using/#:~:text=The%20Difference%20Between%20“Using”%20and%20“By%20Using”,with%20what%20comes%20after%20it.) I have doubts that "by using" is the best option here. Also, ChatGPT suggests that in such contexts just "using" is a better option.

Reply:
There isn't really a reason. I just though it might suit there. Will apply the change